spec/GrumPHP/Process/AsyncProcessRunnerSpec.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace spec\GrumPHP\Process;
+
+use GrumPHP\Configuration\GrumPHP;
+use GrumPHP\Process\AsyncProcessRunner;
+use PhpSpec\ObjectBehavior;
+use Symfony\Component\Process\Process;
+
+/**
+ * @mixin AsyncProcessRunner
+ */
+class AsyncProcessRunnerSpec extends ObjectBehavior
+{
+    public function let(GrumPHP $grumPHP) {
+        $this->beConstructedWith($grumPHP);
+
+        $grumPHP->getProcessAsyncWaitTime()->willReturn(1000);
```

Comment:
Maybe you can set this value to 0 to make sure the tests aren't slowed down by the `usleep()` call?